kimchi/src/circuits/constraints.rs

```diff
@@ -704,9 +704,10 @@ impl<F: PrimeField + SquareRootField> Builder<F> {
         //~ 2. Create a domain for the circuit. That is,
         //~    compute the smallest subgroup of the field that
         //~    has order greater or equal to `n + ZK_ROWS` elements.
-        let domain = EvaluationDomains::<F>::create(gates.len() + ZK_ROWS as usize)?;
+        let domain =
+            EvaluationDomains::<F>::create(gates.len() + (ZK_ROWS + PERM_FINAL_ACC) as usize)?;
 
-        assert!(domain.d1.size > ZK_ROWS);
+        assert!(domain.d1.size > ZK_ROWS + PERM_FINAL_ACC);
```

Comment:
`PERM_FINAL_ACC` should be a permutation-related constant only (see https://www.youtube.com/watch?v=iY2ue8Kfsb0&t=1028s and https://o1-labs.github.io/proof-systems/plonk/zkpm.html)

Reply:
Yes, I just leave this variable in `permutations.rs` file.